shared_directory/Raw_Document_1.json

```diff
@@ -0,0 +1 @@
+{"title": "Raw_Document_1", "content": "We use technology to make New York ready for the future.\n\nWe can adapt to change and unexpected situations.\n\nWe are diverse and agile.\n\nThis is a new change.\n\nThis is another change.\n"}
```

Comment:
This is great!

For the "shared" folder directory structure, I'm thinking we can organize it similar to how smartcat does it-

shared_directory/
    /en
        raw_document_1.json
    /ru
        raw_document_1.json.ru


All of the `en/` files gets populated directly from google drive. And I can setup `serge` to output inside of the each language prefix. Serge _should_ generate those folders for us automatically.

Reply:
@as1729 sounds like a good idea!